generate.go

```diff
@@ -70,6 +131,8 @@ func (c *ConfigGenerator) GenerateAnswers() (Answers, error) {
 		// 2. set service links
 		// note that service link overrides the container link (if the names collide)
 		for key, linkAlias := range clientIpsToServiceLinks[clientIp] {
+			c.SetLinksForRegions(key, linkAlias, cARecs, cCnameRecs, aRegionRecs, cRegionRecs)
```

Comment:
Looks like with the current code, even if the link is considered as global, and setLinksForRegion does something, we still proceed to the line 136 and below?